docs/remote-control/operating-the-remote.md

```diff
@@ -6,29 +6,44 @@ sidebar_label: Operating the Remote
 
 import Keys from '@site/src/components/key.ts';
 
-To connect to the console, open the Titan Remote app and a list of
-available consoles will be shown - hopefully if you have done the
-[network setup](setting-up-the-remote.md) correctly, you will see your console listed.
+The previous section [Setting up the Remote](setting-up-the-remote.md) takes you
+through the steps for getting a mobile device connected to the console.
+
+Once you have a network set up, open the Titan Remote app and a list of
+connected consoles should be shown.
+
+![Titan Remote App connection screen](/docs/images/Remote-Connection-Screen.png)
 
 There is also a local emulator which can be used to test or demonstrate
-the remote when no real console is connected.
+the remote when no real console is connected. If your console does not show
+up in the list, you can enter its IP address manually by clicking the + button 
+in the top right hand corner of the screen.
+
+![Titan Remote Android App with wheels](/docs/images/Remote-Colour-Attributes.png)
 
-![Titan Remote Android App with keypad](/docs/images/Titan-Remote-Android-App-Control-Selection.png)
+The app title bar should be green showing good connection. If the title bar turns
+red then you do not have connection to the console.
```

Comment:
The title bar now has an orange state which it shows up when the connection to the console is slow. We have deemed slow as greater than approximately 250ms to respond to a message.